WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -822,6 +926,43 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
     [self.tableView reloadData];
     [self updateEditButton];
     [self maybeShowNUX];
+    [self maybeShowNoResultsView];
+}
+
+#pragma mark - WPNoResultsViewDelegate
+
+- (void)didTapNoResultsView:(WPNoResultsView *)noResultsView
+{
+    // Find the button in the NoResultsView
+    UIButton *button = nil;
+    for (UIView *subview in noResultsView.subviews) {
```

Comment:
Yeah, I wasn't that keen on doing this here – just wanted to avoid modifying WP-iOS-Shared. But let's do it! https://github.com/wordpress-mobile/WordPress-Shared-iOS/pull/109